src/rtc/h5_prep.py

```diff
@@ -215,36 +234,60 @@ def populate_metadata_group(h5py_obj: h5py.File,
         'identification/boundingPolygon':
             [get_polygon_wkt(burst_in),
             'OGR compatible WKT representation of bounding polygon of the image'],
-        'identification/missionId':
-            [burst_in.platform_id, 'Mission identifier'],
+        # 'identification/missionId':
+        #    [mission_id, 'Mission identifier'],
+        'identification/platformId':
+            [platform_id, 'Platform identifier'],
+        'identification/instrumentName':
+            [platform_id, 'Instrument name'],
         'identification/productType':
             ['RTC-S1', 'Product type'],
-        # NOTE: in NISAR, the value has to be in UPPERCASE or lowercase?
+        'identification/projectName':
+            ['OPERA', 'Project name'],
+        'identification/acquisitionMode':
+            ['Interferometric Wide (IW)', 'Acquisition mode'],
+        'identification/beamID':
+            [beam_id, 'Beam identification (Beam ID)'],
         'identification/lookDirection':
-            ['Right', 'Look direction can be left or right'],
+            ['right', 'Look direction can be left or right'],
         'identification/orbitPassDirection':
-            [burst_in.orbit_direction, 'Orbit direction can be ascending or descending'],
+            [burst_in.orbit_direction.lower(),
+            'Orbit direction can be ascending or descending'],
         # NOTE: using the same date format as `s1_reader.as_datetime()`
         'identification/zeroDopplerStartTime':
-            [burst_in.sensing_start.strftime('%Y-%m-%dT%H:%M:%S.%f'),
+            [burst_in.sensing_start.strftime('%Y-%m-%dT%H:%M:%S.%fZ'),
              'Azimuth start time of product'],
         'identification/zeroDopplerEndTime':
-            [burst_in.sensing_stop.strftime('%Y-%m-%dT%H:%M:%S.%f'),
+            [burst_in.sensing_stop.strftime('%Y-%m-%dT%H:%M:%S.%fZ'),
             'Azimuth stop time of product'],
         'identification/listOfFrequencies':
              [['A'], 'List of frequency layers available in the product'],  # TBC
         'identification/isGeocoded':
             [True, 'Flag to indicate radar geometry or geocoded product'],
+        'identification/productLevel':
+            ['L2', 'Product level'],
+        'identification/productID':
+            [product_id, 'Product identificator'],
+        # 'identification/productSource':
+            # [platform_id, 'Product source'],
         'identification/isUrgentObservation':
             [False, 'List of booleans indicating if datatakes are nominal or urgent'],
         'identification/diagnosticModeFlag':
             [False, 'Indicates if the radar mode is a diagnostic mode or not: True or False'],
         'identification/processingType':
             [processing_type, 'NOMINAL (or) URGENT (or) CUSTOM (or) UNDEFINED'],
+        # datetime format 'YYYY-MM-DD HH:MM:SS'
+        'identification/processingDateTime':
+            [datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
```suggestion
            [datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Reply:
Thank you, @seongsujeong , addressed that in my last commits.